app/src/pages/SectionEdit.tsx

```diff
@@ -45,25 +50,37 @@ type SectionPartial = {
  */
 const SectionEdit = () => {
     const { sid } = useParams();
-    const token = useToken();
+    //const token = useToken();
+    const token = "bruh";
 
     // Component state
     const [section, setSection] = useState<Section>();
     const [exercises, setExercises] = useState<Exercise[]>([]);
+    const [lectures, setLectures] = useState<Lecture[]>([]);
+    const [contentCount, setContentCount] = useState(0);
+
 
-    // Fetch section details
+
+    //Fetch section details
     const { data: sectionData, error: sectionError } = useSWR(
         token ? [sid, token] : null,
         SectionServices.getSectionDetail
     );
 
-    // TODO: This code is a foundation for fetching exercises, will be implemented next PR.
-    // Fetch the exercises data from the server.
-    /*
+   
+
+    // Fetch the exercises data from the server.    
     const { data: exerciseData, error: exerciseError } = useSWR(
-        token ? [`${BACKEND_URL}/exercises/getall/${sid}`, token] : null,
+        token ? [`${BACKEND_URL}/api/exercises/getall/${sid}`, token] : null,
         ExerciseServices.getExerciseDetail
-    );*/
+    );
+    
+    // Fetch the lecture data from the server.    
+    const { data: lectureData, error: lectureError } = useSWR(
+        token ? [`${BACKEND_URL}/api/lectures/getall/${sid}`, token] : null,
```

Comment:
Endpoint does not meet API guidelines.

Reply:
Have changed it from api/lectures/getall/${sid} --> api/lectures/section/${sid}
It's the best solution without changing every get request last minute with the query setup.
There's already one without "getall", which is why there is "section"